src/6model/containers.c

```diff
@@ -545,6 +585,28 @@ static void native_ref_store_i(MVMThreadContext *tc, MVMObject *cont, MVMint64 v
     }
 }
 
+static void native_ref_store_u(MVMThreadContext *tc, MVMObject *cont, MVMuint64 value) {
+    MVMNativeRefREPRData *repr_data = (MVMNativeRefREPRData *)STABLE(cont)->REPR_data;
+    if (repr_data->primitive_type != MVM_STORAGE_SPEC_BP_INT)
+        MVM_exception_throw_adhoc(tc, "This container does not reference a native integer");
+    switch (repr_data->ref_kind) {
+        case MVM_NATIVEREF_LEX:
+            MVM_nativeref_write_lex_i(tc, cont, value);
+            break;
+        case MVM_NATIVEREF_ATTRIBUTE:
+            MVM_nativeref_write_attribute_i(tc, cont, value);
+            break;
+        case MVM_NATIVEREF_POSITIONAL:
+            MVM_nativeref_write_positional_u(tc, cont, value);
```

Comment:
I'd appreciate a comment explaining why this case is `_u`, but the others are `_i`.